chaos-workers/chaos-experiments/scripts/verify-readiness.sh

```diff
@@ -8,13 +8,17 @@ namespace=$(getNamespace)
 # verify that everything is running
 
 if [ "${CHAOS_SETUP}" == "cloud" ]; then
-  # Wait until brokers are ready. The componenet should be broker, but it is wrongly labelled as gateway
+  # Wait until brokers are ready. The component should be broker, but it is wrongly labelled as gateway
   kubectl wait --for=condition=Ready pod -l app.kubernetes.io/component=gateway --timeout=900s -n "$namespace"
-  # Wait until all gateway pods are running.
+  # Wait until all gateway pods are running. We cannot use `--for=condition=Ready` pod for gateway because
+  # this check also includes the evicted pods as long as they are not deleted. Evicted pods are not immediately
+  # deleted by kuberenetes. The following check passes only if all pods are running, and excludes evicted pods.
   kubectl wait --for=condition=Available deployment zeebe-gateway --timeout=120s -n "$namespace"
 else
-  # Wait until brokers are ready. The componenet should be broker, but it is wrongly labelled as gateway
+  # Wait until brokers are ready. The component should be broker, but it is wrongly labelled as gateway
```

Comment:
:upside_down_face: Here the component actually *is* broker so the comment is wrong.

Reply:
In the command, we use the label `app.kubernetes.io/component=gateway` to identify brokers.

Reply:
Sorry. Got it :) It is for the non-cloud deployments :D